clients/admin-ui/src/pages/add-systems/index.tsx

```diff
@@ -1,23 +1,50 @@
 import type { NextPage } from "next";
+import { useEffect } from "react";
 
-import { useAppSelector } from "~/app/hooks";
+import { useAppDispatch, useAppSelector } from "~/app/hooks";
+import { useFeatures } from "~/features/common/features.slice";
 import Header from "~/features/common/Header";
 import Layout from "~/features/common/Layout";
-import { selectStep } from "~/features/config-wizard/config-wizard.slice";
+import {
+  changeStep,
+  selectStep,
+} from "~/features/config-wizard/config-wizard.slice";
 import ConfigWizardWalkthrough from "~/features/config-wizard/ConfigWizardWalkthrough";
 import Setup from "~/features/config-wizard/setup";
 
 const ConfigWizard: NextPage = () => {
+  const dispatch = useAppDispatch();
+  const features = useFeatures();
   const step = useAppSelector(selectStep);
 
-  return step === 0 ? (
-    <Layout title="Config Wizard">
-      <Setup />
-    </Layout>
-  ) : (
+  useEffect(() => {
+    if (features.navV2) {
+      // Add system form
+      dispatch(changeStep(2));
+    }
+  }, [dispatch, features.navV2]);
+
+  return (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
     <>
-      <Header />
-      <ConfigWizardWalkthrough />
+      {features.navV2 ? (
+        <Layout title="Config Wizard">
+          <ConfigWizardWalkthrough />
+        </Layout>
+      ) : (
+        [
+          step === 0 ? (
+            <Layout title="Config Wizard">
+              <Setup />
+            </Layout>
+          ) : (
+            <>
+              <Header />
+              <ConfigWizardWalkthrough />
+            </>
+          ),
+        ]
+      )}
```

Comment:
Not sure why but I get this fun double-page effect 😄 
https://www.loom.com/share/70e5da4e581b47c69b212739f626a6fc